static/js/apps/nl_interface/query_examples.tsx

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Component for a list of example queries
+ */
+
+import _ from "lodash";
+import React from "react";
+
+import { getUrlToken } from "../../utils/url_utils";
+
+// Map of topic to list of example queries
```

Comment:
How did you come up with these queries?  something like "what is the main farming method used" we don't have data for

Reply:
These are just random queries I thought of off the top of my head to show what the UI would look like/what the flow of actions would be. If there are specific queries you want me to take out or add, I can make those changes.